src/cobertura.rs

```diff
@@ -229,12 +231,13 @@ impl ToString for ConditionType {
 }
 
 fn get_coverage(
-    results: CovResultIter,
+    results: &[(PathBuf, PathBuf, CovResult)],
```

Comment:
Could we define this as a type so we don't have to write `&[(PathBuf, PathBuf, CovResult)]` everywhere?

Reply:
pushed